### PR TITLE
feat: headers components

### PR DESCRIPTION
**Pull request by Anderson22-dev**
Criação dos componentes dos headers

### CODE REVIEW

**Review by Eduardo-BarrosoS — CHANGES_REQUESTED** [review]
You should create accordion in another branch

**Comment by qalexandre** [comment]
@Anderson22-dev  faça um pull da develop primeiro, resolva os conflitos e sobe as alterações

File: package.json [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+    "name": "design-systems",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "dev": "next dev",
+        "build": "next build",
+        "start": "next start",
+        "lint": "next lint"
+    },
+    "dependencies": {
```
Comment (Eduardo-BarrosoS) [inline package.json]: Replace fontawesome to lucide react lib

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,13 +1,16 @@
 import { Text } from '@/components/Text'
```
Comment (Eduardo-BarrosoS) [inline src/app/page.tsx]: If you are not using Text component, you can delete this one

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 
 
+import * as React from "react";
+import Header from "@/components/Header";
```
Comment (Eduardo-BarrosoS) [inline src/app/page.tsx:2]: If you are not using React import, you can delete this one

File: src/components/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <div className="flex flex-row items-center">
+            <div className="p-3 pt-0">
+            {back? back: null}
+                <h1 className="w-min text-2xl font-bold text-neutral-800">{title}</h1>
+                <h3 className="text-sm text-neutral-600">{subtitle}</h3>
+            </div>
+            {secondaryActionBtn? secondaryActionBtn: null}
+        </div>
+    </div>
+    <div className="flex flex-row items-center">
```
Comment (Eduardo-BarrosoS) [inline src/components/Header.tsx]: image

You need to add a space between buttons

**Review by Eduardo-BarrosoS — CHANGES_REQUESTED** [review]

File: tailwind.config.ts [hunk 5]
```diff
@@ -121,6 +132,8 @@ const config: Config = {
       }
     },
   },
-  plugins: [],
+  darkMode: "class",
+  
+  plugins: [nextui()],
```
Comment (Eduardo-BarrosoS) [inline tailwind.config.ts]: Are you sure you need this plugin and why?

File: src/components/Header.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from "react";
+
```
Comment (Eduardo-BarrosoS) [inline src/components/Header.tsx:2]: You have to develop pixel perfect what is on figma design.
It means that this component should be responsive.

**Review by Eduardo-BarrosoS — CHANGES_REQUESTED** [review]

**Comment by Eduardo-BarrosoS** [comment]
> You should create accordion in another branch

**Review by Eduardo-BarrosoS — APPROVED** [review]
Approved

**Pull request merged**